docs/pilot.md

```diff
@@ -205,8 +198,14 @@ EESSI currently supports the following HPC applications as well as all their dep
 #### aarch64/arm64
 
   - generic (currently implies `-march=armv8-a` and `-mtune=generic`)
+  - AWS Graviton2
```

Comment:
Why AWS? Is is specific to them?

Reply:
Yes, it's being developed by a company that was acquired by Amazon:
https://en.wikipedia.org/wiki/Annapurna_Labs#AWS_Graviton2
AFAIK they're only available in AWS.